notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<p>"Categorical features are values that can be sorted into categories, such as gender or color." --&gt; "Categorical features have values that are discrete, such as gender or color. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Updated. Thanks!</p>